.changeset/spicy-bottles-obey.md

```diff
@@ -0,0 +1,10 @@
+---
+"@wso2is/admin.validation.v1": minor
+"@wso2is/admin.core.v1": minor
+"@wso2is/myaccount": minor
+"@wso2is/console": minor
+"@wso2is/core": minor
+"@wso2is/i18n": minor
```

Comment:
Since the changes to core and i18n are minor, let's add a separate changeset for those modules with `patch` version upgrade.

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/7185/commits/3f33e369f89ad12cb6c062dc68786697e4a7826e